src/Tree.h

```diff
@@ -81,6 +86,9 @@ class Tree
     bool mActive;
     bool mUserActiveState;
     QString mErrorMessage;
+
+    bool m_isTemporary;
```

Comment:
We would (or I _think_ we would) tend to use `mTemporary` (though I am of the `mIsTemporary` school of thought!) to flag a variable as a class ___member___ {_camelCase_ rather than *underscore\_ space*} now.